README.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Space Travelers' Hub 
+
+> This project is a web app created using React and Redux framework, for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions(the real live data from the SpaceX API).
+
+![image](https://user-images.githubusercontent.com/89970442/152335730-8479c782-c395-4b65-8662-278c3c717e23.png)
+
+![image](https://user-images.githubusercontent.com/89970442/152335792-474269c6-0f8e-4288-984f-1be16c69aaf7.png)
+
+![image](https://user-images.githubusercontent.com/89970442/152335833-77e2270f-2aeb-47ba-bdca-6871a4ccc544.png)
+
+
+## Built With
+
+- React and Redux
+
+
+## Live Demo
+
+[Check the demo here](https://spacehub2022.herokuapp.com/)
+
+## Getting Started
+
```

Comment:
- [ ] Please add the steps about how to clone your repo. Remember to write these steps thinking on someone that doesn't know anything about programming. 💯 😃